07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
+  <head>
+    <title>Profession Portal</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <div id="title">
+      <h1>Profession Portal</h1>
+    </div>
 
-</body>
+    <div id="navigation">
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">Companies</a>
+        <a href="#">Positions</a>
+      </nav>
+    </div>
+
+    <div id="sign-up">
+      <div>
+        <h3>Hire an employee</h3>
+        <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.</p>
+        <input type="submit" value="Sign up as company"/>
```

Comment:
these wouldn't be inputs in the real world, they would be anchors! switch these out when you get a chance. Great work-around though :)